File: webapp-net/SmartTarget/Models/SmartTargetExperiment.cs
using Tridion.SmartTarget.Analytics;
using Tridion.SmartTarget.Query;

namespace Sdl.Web.Modules.SmartTarget.Models
{
    public class SmartTargetExperiment : SmartTargetPromotion
    {
        public ExperimentDimensions ExperimentDimensions { get; set; }

        public ExperimentCookies NewExperimentCookies { get; set; }
    }
}

File: webapp-java/dxa-module-smarttarget/dxa-module-smarttarget-common/src/main/java/com/sdl/dxa/modules/smarttarget/model/entity/SmartTargetExperiment.java
package com.sdl.dxa.modules.smarttarget.model.entity;

import com.tridion.smarttarget.analytics.tracking.ExperimentDimensions;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * XO Experiment.
 */
@EqualsAndHashCode(callSuper = true)
@Data
public class SmartTargetExperiment extends SmartTargetPromotion {
    private final ExperimentDimensions experimentDimensions;
}
